drivers/ethernet/eth_mcux.c

```diff
@@ -932,6 +951,30 @@ static void eth_mcux_ptp_isr(void *p)
 }
 #endif
 
+#ifdef CONFIG_SOC_SERIES_IMX_RT
```

Comment:
I'd like to avoid direct dependencies on the SoC like this. If you add `interrupt-names = "COMMON"` to the dts, you can make this condition `#ifdef ETH_IRQ_COMMON` instead.